### ISSUE TITLE
sqlite error when running ????? syncdb

### ISSUE THREAD

**Issue by Marc-Bellario** [gadio/moma-django#5]
when I run the sample it will not syncdb - also momadjango.org is likewise failing on registration
with an sqlite error

for the syncdb: I get the following:

Traceback (most recent call last):
  File "manage.py", line 25, in <module>
    execute_from_command_line(sys.argv)
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\core\management__init__.
py", line 443, in execute_from_command_line
    utility.execute()
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\core\management__init__.
py", line 382, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\core\management\base.py",
 line 196, in run_from_argv
    self.execute(_args, *_options.**dict**)
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\core\management\base.py",
 line 232, in execute
    output = self.handle(_args, *_options)
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\core\management\base.py",
 line 371, in handle
    return self.handle_noargs(**options)
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\core\management\commands\
syncdb.py", line 57, in handle_noargs
    cursor = connection.cursor()
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\db\backends__init__.py",
 line 317, in cursor
    cursor = self.make_debug_cursor(self._cursor())
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\db\backends\sqlite3\base.
py", line 288, in _cursor
    self._sqlite_create_connection()
  File "C:\Users\Marc\virtual_main\prj14_moma_django\lib\site-packages\django\db\backends\sqlite3\base.
py", line 278, in _sqlite_create_connection
    self.connection = Database.connect(**kwargs)
sqlite3.OperationalError: unable to open database file

**Comment by gadio** [comment]
You are correct (sorry - I just noticed that post). I fixed the example web site and it is working now.
With regards to the sqlite - you should have a database file created under the moma_example directory. Do you have that file there?
